SAGE/Games/Kane's Wrath/GameObject.xml

```diff
@@ -1298,78 +1094,24 @@
 	</Asset>
 
 	<Asset
-		id="AutoResolveArmor">
+		id="AutoResolveInfo">
+		<Entry
+			id="AutoResolveUnitType"
+			AssetType="String"
+			IsAttribute="true" />
+		<EntryPoid
+			id="AutoResolveBody"
+			IsAttribute="true" />
 		<EntryReference
-			id="Armor"
+			id="AutoResolveArmor"
 			AssetType="ArmorTemplate"
 			IsAttribute="true" />
-		<EntryList
-			id="RequiredUpgrade"
-			AssetType="UpgradeTemplateWeakRef"
-			MinLength="0"
-			MaxLength="0" />
-		<EntryList
-			id="ExcludedUpgrade"
-			AssetType="UpgradeTemplateWeakRef"
-			MinLength="0"
-			MaxLength="0" />
-	</Asset>
-
-	<Asset
-		id="AutoResolveWeapon">
 		<EntryReference
-			id="Weapon"
-			AssetType="WeaponTemplate"
+			id="AutoResolveWeapon"
+			AssetType="ArmorTemplate"
```

Comment:
This should be a WeaponTemplate.
Also shuffling around stuff is really messing with diffing.

Reply:
I would like to follow the same ordering as the EA Schemas. Perhaps I make 2 commits, the first one will be any edits and additions to all Asset Types and the second one just reshuffles the GameObject file

Reply:
If you mean 2 prs then yes, that'd be great, as it'd be a lot easier to compare.

Reply:
I've unshuffled it and fixed the issue